docs/extensions/readme.md

```diff
@@ -2,7 +2,13 @@
 
 ### Introduction
 
-The following documents describes AutoRest specific vendor extensions for [OpenAPI 2.0](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md) schema. Some of the extensions are only applicable to Microsoft Azure and as such are only available in Azure code generators (e.g. Azure.CSharp, Azure.NodeJS, etc.).
+The following documents describes AutoRest specific vendor extensions for [OpenAPI 2.0](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md) schema. Some of the extensions are only metadatas and would not trigger code generator update, some are applicable to ARM APIs only. Code generation impact and support may vary per language.
+
+### Metadata extensions
```

Comment:
Can we put metadata extension last. Feels weird that we'd start describing extension that have no meaning inside autorest.

Reply:
Sure, I had the same thought too. I'll change accordingly after this PR: https://github.com/Azure/autorest/pull/4365/files